db/migrate/20150504112462_change_collation_for_tag_names.acts_as_taggable_on_engine.rb

```diff
@@ -0,0 +1,10 @@
+# This migration comes from acts_as_taggable_on_engine (originally 5)
+# This migration is added to circumvent issue #623 and have special characters
+# work properly
+class ChangeCollationForTagNames < ActiveRecord::Migration
+  def up
+    if ActsAsTaggableOn::Utils.using_mysql?
+      execute("ALTER TABLE tags MODIFY name varchar(255) CHARACTER SET utf8 COLLATE utf8_bin;")
```

Comment:
Line is too long. [95/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.